library/core/src/future/ready.rs

```diff
@@ -34,17 +34,16 @@ impl<T> Ready<T> {
     /// # Examples
     ///
     /// ```
-    /// #![feature(ready_into_inner)]
     /// use std::future;
     ///
     /// let a = future::ready(1);
-    /// assert_eq!(a.into_inner(), 1);
+    /// assert_eq!(a.unwrap(), 1);
     /// ```
-    #[unstable(feature = "ready_into_inner", issue = "101196")]
+    #[stable(feature = "ready_into_inner", since = "CURRENT_RUSTC_VERSION")]
```

Comment:
I'm not entirely sure what the purpose is of these feature names after stabilization.
Should I rename it to `ready_unwrap` as well?

Reply:
The purpose is that the compiler can instruct the user what to do with their pre-existing `#![feature(...)]` attribute.

If you rename the feature, the compiler will tell them "I have no idea what this feature means." https://play.rust-lang.org/?version=nightly&mode=debug&edition=2021&gist=85bfc71771a9a86d0f071a13bbee5436

If you do not rename the feature, the compiler will tell them "this feature has been stabilized since 1.7X.0 so please remove this feature attribute." https://play.rust-lang.org/?version=nightly&mode=debug&edition=2021&gist=756ce3bcaad3211a8520fb100a8efc22

Reply:
Alright, thank you!